service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapper.java

```diff
@@ -80,6 +82,12 @@ public String mapObservationToObservationStatement(Observation observation, bool
             );
         }
 
+        if (observation.hasPerformer()) {
+            final String participantReference = idMapper.getOrNew(observation.getPerformerFirstRep());
+            final String participantBlock = participantMapper.mapToParticipant(participantReference, "PPRF");
```

Comment:
These codes could be captured in a common place. Only "PPRF" for this AC, but more will be introduced by the others.

Reply:
The spreadsheet mentions `typeCode` "PRF", not "PPRF"

Reply:
Hah! The example on the ticket has PPRF, but we'll take the spreadsheet as the source of truth

Reply:
Please confirm with @dmcavenue1 which one is correct if there is any doubt

Reply:
You could probably find the codes in the MiM also

Reply:
It is "PRF" for Performer by the MiM https://data.developer.nhs.uk/dms/mim/6.3.01/Vocabulary/x_RelAgentType.htm

Reply:
We do use PPRF in other contexts but yes it should be PRF here